client/js/commands/join.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const utils = require("../utils");
+
+exports.input = function(args) {
+	const channel = args[0];
+
+	if (channel) {
+		const chan = utils.findCurrentNetworkChan(channel);
+
+		if (chan.length) {
+			chan.trigger("click");
+			return true; // we're done here, server doesn't need to do anything.
```

Comment:
Actually, this again changed behaviour from what it is on master, and we do always pass `/join` command to the server.

So just remove both `return` in this function.

Reply:
Fixed